src/FormatMetadata.cpp

```diff
@@ -0,0 +1,57 @@
+// Chemfiles, a modern library for chemistry file reading and writing
+// Copyright (C) Guillaume Fraux and contributors -- BSD license
+
+#include "chemfiles/FormatMetadata.hpp"
+
+#include "chemfiles/utils.hpp"
+#include "chemfiles/error_fmt.hpp"
+using namespace chemfiles;
+
+static void check_not_empty(const char* value, string_view field, const char* name) {
```

Comment:
At first glance the parameter names are rather non-descriptive and I got confused. Do you think `field_value`, `field_name`, and `format_name` are better suited?